src/legacy/core_plugins/inspector_views/public/requests/request_selector.js

```diff
@@ -137,32 +155,46 @@ class RequestSelector extends Component {
           { selectedRequest.status !== RequestStatus.PENDING &&
             <EuiToolTip
               position="left"
-              title={selectedRequest.status === RequestStatus.OK ? 'Request succeeded' : 'Request failed'}
-              content="The total time the request took."
+              title={selectedRequest.status === RequestStatus.OK ?
+                <FormattedMessage
+                  id="inspectorViews.requests.requestSucceededTooltipTitle"
+                  defaultMessage="Request succeeded"
+                /> :
+                <FormattedMessage
+                  id="inspectorViews.requests.requestFailedTooltipTitle"
+                  defaultMessage="Request failed"
+                />
+              }
+              content={<FormattedMessage
+                id="inspectorViews.requests.requestTooltipDescription"
+                defaultMessage="The total time the request took."
+              />}
             >
               <EuiBadge
                 color={selectedRequest.status === RequestStatus.OK ? 'secondary' : 'danger'}
                 iconType={selectedRequest.status === RequestStatus.OK ? 'check' : 'cross'}
               >
-                {selectedRequest.time}ms
+
+                {selectedRequest.time}<FormattedMessage
```

Comment:
The time should be part of the string as a value, so you're having `{requestTime}ms` in the string.